app/controllers/experiments_controller.rb

```diff
@@ -237,49 +239,25 @@ def move_modal
 
   # POST: move_experiment(id)
   def move
-    project = Project.find_by_id(params[:experiment].try(:[], :project_id))
-    old_project = @experiment.project
-
-    # Try to move the experiment
-    success = true
-    if @experiment.moveable_projects(current_user).include?(project)
-      success = @experiment.move_to_project(project)
-    else
-      success = false
-    end
+    project = Project.find params[:experiment][:project_id]
```

Comment:
Previously a `try` block was used. Is there any particular reason you removed this?

Reply:
I thought we always have "experiment" and "project_id" or?

Reply:
I'd assume this catch was used in case someone forges request on JS side.

Reply:
In that case Urban's code right on the spot. If request comes without experiment param it should fail miserably. 

Actually the lack of experiment param should be caught by strong_params and throw the error.